### PR TITLE
Add security contact for plugins where Modern Platform team is involved

### PR DESCRIPTION
**Pull request by Vlatombe**
Pinging maintainers for each involved plugin

Updating security contact. This is the same CloudBees team behind as
before
* `artifact-manager-s3` (@jglick )
* `kubernetes-credentials` (@maxlaverse)

New security contact
* `aws-global-configuration` (@jglick)
* `aws-java-sdk` (@Vlatombe)
* `kubernetes` (@Vlatombe)
* `one-shot-executor` (no current maintainer, I can volunteer if needed)
* `saml` (@kuisathaverat)
* `view-job-filters` (@jglick)

Removing our contact from mesos plugin. We do no longer maintain it.

# Description

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [ ] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [ ] Make sure the file is created in `permissions/` directory
- [ ] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [ ] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [ ] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [ ] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [ ] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [ ] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [ ] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

File: permissions/plugin-one-shot-executor.yml
Comment (jglick) [inline permissions/plugin-one-shot-executor.yml:5]: Is this plugin still in use?

**Review by jglick — APPROVED** [review]

**Comment by slide** [comment]
@daniel-beck for security contact changes sign-off

File: permissions/plugin-one-shot-executor.yml
Comment (jglick) [reply permissions/plugin-one-shot-executor.yml:5]: Evidently it is, a bit.

**Review by daniel-beck — APPROVED** [review]
Security contact approval from officer. Thanks for the ping!

**Comment by slide** [comment]
Still waiting on approval from @kuisathaverat

File: permissions/plugin-one-shot-executor.yml
Comment (daniel-beck) [inline permissions/plugin-one-shot-executor.yml:6]: This is weird.

**Review by kuisathaverat — APPROVED** [review]

File: permissions/plugin-kubernetes-credentials.yml
Comment (daniel-beck) [inline permissions/plugin-kubernetes-credentials.yml:9]: Still need approval here.

Comment (maxlaverse) [reply permissions/plugin-kubernetes-credentials.yml:9]: Do you need the approval from all of us or is mine enough ? Else I can contact him

Comment (timja) [reply permissions/plugin-kubernetes-credentials.yml:9]: Just need one

Comment (timja) [reply permissions/plugin-kubernetes-credentials.yml:9]: @maxlaverse do you approve this?

Comment (maxlaverse) [reply permissions/plugin-kubernetes-credentials.yml:9]: Yes, I do approve this update of security contact

**Pull request merged**
